aggregation/src/main/java/com/study/aggregation/AggregationApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.study.aggregation;
+
+import com.study.controller.AggregationController;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackageClasses = AggregationController.class)
```

Comment:
If you really want to ComponentScan, you can use `@SpringBootApplication(ComponentScan=xx)`